### ISSUE TITLE
check table is empty before do import

### ISSUE THREAD

**Issue by july2993** [pingcap/br#995]
Please answer these questions before submitting your issue. Thanks!

1. What did you do?
If possible, provide a recipe for reproducing the error.
```
/*!40101 SET NAMES binary*/;
CREATE TABLE `test1` (
  `a` int(11) DEFAULT NULL
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin;
```
import the table using lighting with local backend.

run flowing SQL
```
rename test1 to test1_back;
delete from test1;
insert into test1 (select * from test1_back)
```
import again failed by checksum mismatch (although the data exist in table equals the source)

for local and importer backend, lighting assumes the imported data is empty.
if the user DO NOT know lighting can only init an empty table, it may
corrupt the table, also, will fail to pass checksum after import.

2. What did you expect to see?
check table is empty before do import data.
if it is not empty, print the related error to force the user to clean up the table if the user wants to import the table.

3. What did you see instead?
for local/importer backend, lighting assumes the imported table should be empty but it does not check it.
the checksum will fail after import data, may even corrupt the origin table, like failed to import and data/index not consistent, and there's no way to clean up the only imported data.

4. What version of BR and TiDB/TiKV/PD are you using?

5. Operation logs
   - Please upload `br.log` for BR if possible
   - Please upload `tidb-lightning.log` for TiDB-Lightning if possible
   - Please upload `tikv-importer.log` from TiKV-Importer if possible
   - Other interesting logs

6. Configuration of the cluster and the task
   - `tidb-lightning.toml` for TiDB-Lightning if possible
   - `tikv-importer.toml` for TiKV-Importer if possible
   - `topology.yml` if deployed by TiUP

7. Screenshot/exported-PDF of Grafana dashboard or metrics' graph in Prometheus if possible

**Comment by july2993** [comment]
fix by #967 #985

**Comment by kennytm** [comment]
@july2993 the two PRs are merged can we close this issue?

**Comment by july2993** [comment]
> @july2993 the two PRs are merged can we close this issue?

sure
fix in 5.0.1 & 4.0.13
